src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationServiceTests.cs

```diff
@@ -32,6 +32,948 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
 {
     public class PublicationServiceTests
     {
+        [Fact]
+        public async Task GetMyPublicationsAndReleasesByTopic_CanViewAllReleases_ReleaseOrder()
+        {
+            var topic = new Topic
+            {
+                Id = Guid.NewGuid(),
+            };
+
+            // The order they should appear in result
```

Comment:
Might be worth saying in this comment what drives this order, as for me as a casual reader of this test, I don't know why it should be this order :)